analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -195,7 +196,20 @@ abstract class PackageManager {
                 } catch (e: Exception) {
                     e.showStackTrace()
 
-                    result[definitionFile] = ProjectAnalyzerResult(Main.allowDynamicVersions, Project.EMPTY,
+                    val definitionFilePath = VersionControlSystem.getPathToRoot(definitionFile) ?: ""
+
+                    val errorProject = Project.EMPTY.copy(
+                            id = Identifier(
+                                    provider = Unmanaged.toString(),
```

Comment:
I'm not sure we should use `Unmanaged` here. We *do* recognize after all which package manager this is *supposed* to be. Can't we use the name of the package manager that failed? That is, in the expected output above

    id: "Unmanaged::analyzer/src/funTest/assets/projects/synthetic/npm/multiple-lockfiles/package.json:"

should IMO read

    id: "NPM::analyzer/src/funTest/assets/projects/synthetic/npm/multiple-lockfiles/package.json:"

Reply:
Done. I have added another commit to be able to get the name from the `PackageManager` class.